fre/make/createCompile.py

```diff
@@ -6,7 +6,7 @@
 from pathlib import Path
 from multiprocessing.dummy import Pool
 import click
-from .gfdlfremake import varsfre, platformfre, yamlfre, targetfre, buildBaremetal
+from .gfdlfremake import varsfre, yamlfre, targetfre, buildBaremetal
```

Comment:
I'm not sure about this. One of the many reasons we need to make some tests for fre make, to see if this still works. I think there's a bit in this script that references `.hasPlatforms` which I believe is a function in the `platforms` class in `platformfre.py`

Reply:
Ok yeah, you can disregard this comment. This line in fremake that uses the `platforms` class and `hasPlatform` function 
```
if modelYaml.platforms.hasPlatform(platformName):
```
is an object (modelYaml) created using `yamlfre` (which uses platformfre). We're all good here!